app/src/main/java/com/example/tasky/agenda/presentation/itemdetail/viewmodel/EventDetailViewModel.kt

```diff
@@ -8,17 +8,20 @@ import androidx.lifecycle.ViewModel
 import androidx.lifecycle.viewModelScope
 import com.example.tasky.TaskyRoutes
 import com.example.tasky.agenda.domain.EventRepository
+import com.example.tasky.agenda.domain.model.Attendee
 import com.example.tasky.agenda.domain.util.toLocalDate
 import com.example.tasky.agenda.domain.util.toLocalTime
 import com.example.tasky.agenda.domain.util.toLong
+import com.example.tasky.authentication.domain.EmailValidator
```

Comment:
Since you´ll also use email validator here (and on auth). Maybe it can be added to Core